src/index.html

```diff
@@ -6,7 +6,434 @@
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <img
+      src="images/logo.png"
+      alt="header icon"
+      class="header__icon"
+      >
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#">APPLE</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">SAMSUNG</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">СМАРТФОНЫ И ТЕЛЕФОНЫ</a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              data-qa="nav-hover"
+              href="#"
+            >
+              НОУТБУКИ И КОМПЬЮТЕРЫ
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">ГАДЖЕТЫ</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">ПЛАНШЕТЫ</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">ФОТО</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">ВИДЕО</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main-container main-container--page">
+      <div class="card card--page" data-qa="card">
+        <img
+          src="images/imac.jpeg"
+          alt="Imac"
+          class="card__image"
+        >
+
+        <p class="card__product-name">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </p>
+
+        <p class="card__product-code">
+          Код товара: 195434
+        </p>
+
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__reviews">
+            Отзывов: 5
+          </p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-text">
+            Цена:
+          </p>
+
+          <p class="card__price-value">
+            69 999 грн
+          </p>
+        </div>
+
+        <a
+          href="#"
+          class="card__buy-link"
+          data-qa="card-hover"
+        >
+          Купить
+        </a>
+      </div>
+
+      <div class="card card--page" data-qa="card">
```

Comment:
Don't set data-qa="card" and data-qa="card-hover" for all cards. You should set them only for the first card.

Reply:
Fix this and restart tests and then redeploy your project

Reply:
Pay attention to task requirements:
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/52370890/169232054-0f0fcf7c-3e29-4094-bd4b-ed0e243af5e3.png">